x/il/keeper/abci.go

```diff
@@ -0,0 +1,78 @@
+package keeper
+
+import (
+	"time"
+
+	"github.com/armon/go-metrics"
+
+	"github.com/cosmos/cosmos-sdk/telemetry"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	peggytypes "github.com/althea-net/peggy/module/x/peggy/types"
+	"github.com/peggyjv/sommelier/x/il/types"
+)
+
+// EndBlocker is called at the end of every block
+func (k Keeper) EndBlocker(ctx sdk.Context) {
+	defer telemetry.ModuleMeasureSince(types.ModuleName, time.Now(), telemetry.MetricKeyEndBlocker)
+
+	params := k.GetParams(ctx)
+
+	k.IterateStoplossPositions(ctx, func(address sdk.AccAddress, stoploss types.Stoploss) (stop bool) {
+		//   TODO: fetch pair info from oracle given the pair address
```

Comment:
Should I add a helper function for this to the oracle, or are you OK pulling the whole blob up at the beginning of end blocker?